Makefile

```diff
@@ -238,54 +101,71 @@ update_devworkspace_api:
 ### update_devworkspace_crds: pull latest devworkspace CRDs to ./devworkspace-crds. Note: pulls master branch
 update_devworkspace_crds:
 	./update_devworkspace_crds.sh $(DEVWORKSPACE_API_VERSION)
-
-
-### local: set up cluster for local development
-local: _print_vars _set_ctx _create_namespace _deploy_registry _set_registry_url _update_yamls _update_crds _update_controller_deps _reset_yamls _reset_ctx
-
-### generate: generates CRDs and Kubernetes code for custom resource
-generate:
-ifeq ($(shell operator-sdk version | cut -d , -f 1 | cut -d : -f 2 | cut -d \" -f 2),$(OPERATOR_SDK_VERSION))
-	operator-sdk generate k8s
-	operator-sdk generate crds
-	patch/patch_crds.sh
+###### End rules for dealing with devfile/api
+
+### test: Run tests
+ENVTEST_ASSETS_DIR = $(shell pwd)/testbin
+test: generate fmt vet manifests
+	mkdir -p $(ENVTEST_ASSETS_DIR)
+	test -f $(ENVTEST_ASSETS_DIR)/setup-envtest.sh || curl -sSLo $(ENVTEST_ASSETS_DIR)/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.6.3/hack/setup-envtest.sh
+	source $(ENVTEST_ASSETS_DIR)/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
+
+### manager: Build manager binary
+manager: generate fmt vet
+	go build -o bin/manager main.go
+
+### run: Run against the configured Kubernetes cluster in ~/.kube/config
+run: _generate_related_images_env
+	source $(RELATED_IMAGES_FILE)
+	go run ./main.go
+
+debug: _generate_related_images_env
+	source $(RELATED_IMAGES_FILE)
+	dlv debug --listen=:2345 --headless=true --api-version=2 ./main.go --
+
+### install: Install CRDs into a cluster
+install: manifests _kustomize _create_namespace
+	$(KUSTOMIZE) build config/crd | kubectl apply -f -
+
+#### uninstall: Uninstall CRDs from a cluster
+uninstall: manifests _kustomize
```

Comment:
previously it used to remove controller as well. What is rule to remove controller ?